cpp/conv_templates.cc

```diff
@@ -521,9 +546,23 @@ Conversation GLM() {
 
 }  // namespace
 
+bool containsBothWordsInRWKVWorld(const std::string& str) {
+    std::string lowerStr = str;
+    std::transform(str.begin(), str.end(), lowerStr.begin(), ::tolower);
+
+    return (lowerStr.find("rwkv") != std::string::npos) && 
+           (lowerStr.find("world") != std::string::npos);
+}
+
 using ConvFactory = Conversation (*)();
 
-Conversation Conversation::FromTemplate(const std::string& name) {
+Conversation Conversation::FromTemplate(const std::string& name, const std::string& model_name) {
```

Comment:
Due to the differences in the rwkv world models, such as rwkv-4-world and rwkv-5-world, compatibility has been implemented here.